framework/db/ActiveQuery.php

```diff
@@ -310,6 +316,32 @@ public function one($db = null)
         return null;
     }
 
+    /**
+     * {@inheritdoc}
+     *
+     * @return T[][]
+     * @psalm-return T[][]|BatchQueryResult
+     * @phpstan-return T[][]|BatchQueryResult
+     * @codeCoverageIgnore
+     */
+    public function batch($batchSize = 100, $db = null)
+    {
+        return parent::batch($batchSize, $db);
+    }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @return T[]
+     * @psalm-return T[]|BatchQueryResult
+     * @phpstan-return T[]|BatchQueryResult
+     * @codeCoverageIgnore
+     */
+    public function each($batchSize = 100, $db = null)
+    {
+        return parent::each($batchSize, $db);
+    }
```

Comment:
Can we do it without adding methods? I think `@method` annotation for a class may work.

Reply:
Unfortunately, `@method` phpdoc doesn't support multiple `@return` phpdocs